v0/pipeline/steps/cd/aws-cdk-deploy-step-info.yaml

```diff
@@ -0,0 +1,116 @@
+title: AwsCdkDeployStepInfo
```

Comment:
move this file to common module steps/common as this is being referred by multiple modules

Reply:
should we do that for other  steps also?
If we would look at v0/pipeline/stages/cf/execution-wrapper-config.yaml there are a lot of stages from cd and custom

Reply:
Done